src/reducers/sprintBacklogReducer.ts

```diff
@@ -182,6 +284,23 @@ export const sprintBacklogReducer = (
                 }
                 return;
             }
+            case ActionTypes.API_ADD_SPRINT_BACKLOG_ITEM_PART_REQUEST: {
+                draft.splitInProgress = true;
+                return;
+            }
+            // TODO: Add something similar for openingDetailMenuBacklogItemId when API calls fail
+            case ActionTypes.API_ADD_SPRINT_BACKLOG_ITEM_PART_SUCCESS: {
+                draft.splitInProgress = false;
+                draft.openedDetailMenuBacklogItemId = null;
+                return;
+            }
+            case ActionTypes.API_ADD_SPRINT_BACKLOG_ITEM_PART_FAILURE: {
+                draft.splitInProgress = false;
+                draft.openedDetailMenuBacklogItemId = null;
+                return;
+            }
+            // TODO: Add something similar for openingDetailMenuBacklogItemId??? Although I think this will be handled when API call
```

Comment:
Look into this

Reply:
any of the api calls that are made could fail... if they fail, then openingDetailMenuBacklogItemId will need to be cleared like this